apps/tasks/lib/main.dart

```diff
@@ -48,8 +48,9 @@ class MyApp extends StatelessWidget {
           ],
           supportedLocales: getSupportedLocals(),
           locale: Locale(languageNotifier.language),
-          home: const SafeArea(
-            child: LoginScreen(),
+          home: Container(
+            color: themeNotifier.getIsDarkNullSafe(context) ? Colors.white10 : Colors.white,
```

Comment:
```suggestion
            color: Theme.of(context).scaffoldBackgroundColor
```
Prefer using the background color somewhere from the theme

Reply:
I've reverted this right now. Seams like that `. scaffoldBackgroundColor` always holds the value of the light theme. Any idea why?

![IMG_1589](https://github.com/helpwave/mobile-app/assets/40192999/de81a7a2-9796-4033-bc1f-5e96b50fe829)